site/docs.html

```diff
@@ -12,6 +12,19 @@
    }
    ul.doclist {
      font-size: 18px;
+     list-style-type: none;
+     padding-left: 1em;
+     padding-right: 1em;
+   }
+   li {
+     border-top: 1px solid #ddd;
+     padding: 0.12em;
+   }
+   li.first {
```

Comment:
IMO better to do li:first-child here, and then you can leave off class="first" below. This keeps the styling separate from the list structure.

Reply:
oh very good, I did not know about that.